domain/src/main/java/com/bottlerocketstudios/brarchitecture/domain/models/Workspace.kt

```diff
@@ -0,0 +1,7 @@
+package com.bottlerocketstudios.brarchitecture.domain.models
+
+data class Workspace(
+    val slug: String?,
+    val name: String?,
+    val uuid: String?
```

Comment:
@VanyaHuamanBottleRocket nit: Add trailing comma for less noisy future diffs :)

```suggestion
    val uuid: String?,
```